src/base/object_representation.h

```diff
@@ -679,35 +684,22 @@
       }	\
     else \
       {	\
-	OR_GET_BIGINT (((char *) (ptr)) + OR_REP_OFFSET + OR_MVCC_REP_SIZE, (valp)); \
+	  OR_GET_BIGINT (((char *) (ptr)) + OR_MVCC_INSERT_ID_OFFSET, (valp)); \
```

Comment:
It would be nice to have the same structure for `OR_GET_MVCC_INSERT_ID` and `OR_GET_MVCC_DELETE_ID` :smiley: